lib/livebook_web/live/settings_live.ex

```diff
@@ -65,22 +65,25 @@ defmodule LivebookWeb.SettingsLive do
                 </.labeled_text>
               </div>
 
-              <%= live_redirect to: Routes.live_dashboard_path(@socket, :home),
-                                  class: "button-base button-outlined-gray" do %>
-                <.remix_icon icon="dashboard-2-line" class="align-middle mr-1" />
-                <span>Open dashboard</span>
-              <% end %>
+              <div class="self-center">
+                <%= live_redirect to: Routes.live_dashboard_path(@socket, :home),
+                                    class: "button-base button-outlined-gray" do %>
+                  <.remix_icon icon="dashboard-2-line" class="align-middle mr-1" />
+                  <span>Open dashboard</span>
+                <% end %>
+              </div>
             </div>
           </div>
           <!-- Updates -->
           <div class="flex flex-col space-y-4">
             <h2 class="text-xl text-gray-800 font-semibold pb-2 border-b border-gray-200">
               Updates
             </h2>
-            <form class="mt-4" phx-change="save" onsubmit="return false;">
+            <form class="flex justify-between mt-4" phx-change="save" onsubmit="return false;">
+              <span class="text-gray-700">Show banner when a new version of Livebook is available</span>
               <.switch_checkbox
                 name="update_check_enabled"
-                label="Show banner when a new Livebook version is available"
+                label=""
```

Comment:
We should improve responsiveness inside the `<.switch_checkbox>` component directly, here:

https://github.com/livebook-dev/livebook/blob/e9e0bda94f01f6d21e497bcc64c9ea7cd4156102/lib/livebook_web/live/live_helpers.ex#L293-L309

Reply:
Looks like the difference is `space-x-3`, on mobile we can make it `space-x-1` or 0 if it still looks bad.

Reply:
@jonatanklosko so you mean we should keep the labels, right?

Reply:
That's what I'm thinking yeah.